scripts/code_replacements/rename_constants_bwfile.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Why are you committing this file?

After it's run once to produce this commit, it's not going to be used.
Could you put the script in the commit message instead?

Reply:
Also, you can make the script shorter by doing "find sbws tests …"